cdap-runtime-ext-remote-hadoop/src/main/resources/remote-hadoop.json

```diff
@@ -0,0 +1,49 @@
+{
+  "configuration-groups": [
+    {
+      "label": "SSH Information",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Host",
+          "name": "host",
+          "required": true,
+          "description": "The edge node to connect to when launching jobs.",
+          "widget-attributes": {
+            "placeholder": "Specify the host to connect to"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "User",
+          "name": "user",
+          "required": true,
+          "description": "The user to SSH to the cluster as.",
+          "widget-attributes": {
+            "placeholder": "Specify your User ID"
+          }
+        },
+        {
+          "widget-type": "textarea",
+          "label": "SSH Public Key",
```

Comment:
why does the user need to configure the public key?

Reply:
In order to ssh into a machine, you don't need the public key, right?

Reply:
right, you don't. I thought SSHContext.createSSHSession() was using it somehow but I see now it just uses the 'user' field in SSHPublicKey

Reply:
Actually, it does seem to be used here:
https://github.com/caskdata/cdap/blob/release/5.0/cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/DefaultSSHContext.java#L88

Though, technically, for the remote hadoop provisioner, the value is written to a file there, but not actually ever used.

So, we can probably use an empty string, but not null.